model/event/BatteryBeforeRemoveEvent.php

```diff
@@ -19,6 +19,29 @@
 
 namespace oat\taoBattery\model\event;
 
-class BatteryBeforeRemoveEvent extends AbstractBatteryEvent
+class BatteryBeforeRemoveEvent
```

Comment:
Do we ever need to keep this class? As far as I can see, It's not used anymore. So maybe delete one or mark as deprecated?

Reply:
This is used in rework assignment branch

Reply:
then should we still trigger the event? or maybe 2 events at a time - both BatteryBeforeRemoveEvent (for back compatibility) & BatteryRemovedEvent (for it to be logged)